apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -28,8 +30,20 @@ function SectionProgressSelector({
   }
 
   const toggleV1OrV2Button = () => (
-    <div>
-      <Button onClick={onShowProgressTableV2Change}>Toggle V1/V2</Button>
+    <div style={{float: 'right'}}>
+      <Button
```

Comment:
I added this as a button just for expediency, but I don't think there's any reason it needs to be. I think it probably fits better as a link (probably u[sing the design component](https://github.com/code-dot-org/code-dot-org/tree/staging/apps/src/componentLibrary/link) ([storybook](http://localhost:9001/?path=/story/designsystem-link-component--default-link)))

Reply:
Another thing to consider is where we want to add this as a floating CSS component on the selector or whether we want to add them to each version directly (e.g. add to SectionProgressV2.jsx). The first version lets us have one component to debug, but the second gives us more control over style and avoids using 'float' which can be a nuisance

Reply:
I like the idea of the Link instead!  I changed that!

I personally like having it housed in a different component (`SectionProgressSelector`) just to make `SectionProgresV2` a bit cleaner.  That being said, I don't think I have experience with the pain of using a `float` yet.  So I see two options:

1) Leave it as is in `SectionProgressSelector` knowing and noting that we could easily change it in the future (this is what I did here)
2) Move it to the `SectionProgressV2` component in anticipation that float will be messy in the future. 

Let me know your thoughts!